cli-windows/set-dockernat-networkategory-to-private.ps1

```diff
@@ -0,0 +1 @@
+﻿Get-NetConnectionProfile | Where-Object { $_.InterfaceAlias -match "(DockerNAT)" } | ForEach-Object { Set-NetConnectionProfile -InterfaceIndex $_.InterfaceIndex -NetworkCategory Private }
```

Comment:
I think it'd be great to include a comment stating this needs to run in an elevated session.

Reply:
I have updated the script to Have #Requires -RunAsAdministrator at the start, so it provides clear message when someone executes with without elevated privilege. 

.\set-dockernat-networkategory-to-private.ps1 : The script 'set-dockernat-networkategory-to-private.ps1' cannot be run
because it contains a "#requires" statement for running as Administrator. The current Windows PowerShell session is
not running as Administrator. Start Windows PowerShell by  using the Run as Administrator option, and then try running
the script again.
At line:1 char:1
+ .\set-dockernat-networkategory-to-private.ps1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (set-dockernat-n...-to-private.ps1:String) [], ScriptRequiresException
    + FullyQualifiedErrorId : ScriptRequiresElevation